core/store/src/db.rs

```diff
@@ -49,22 +49,66 @@ impl Into<io::Error> for DBError {
 #[derive(PartialEq, Debug, Copy, Clone, EnumIter, BorshDeserialize, BorshSerialize, Hash, Eq)]
 pub enum DBCol {
     /// Column to indicate which version of database this is.
+    /// - *Rows*: single row [VERSION_KEY]
+    /// - *Content type*: The version of the database (u32), serialized as JSON.
     ColDbVersion = 0,
+    /// Column that store Misc cells.
+    /// - *Rows*: multiple, for example "GENESIS_JSON_HASH", "HEAD_KEY", [LATEST_KNOWN_KEY] etc.
+    /// - *Content type*: cell specific.
     ColBlockMisc = 1,
+    /// Column that stores Block content.
+    /// - *Rows*: block hash (CryptHash)
+    /// - *Content type*: [near_primitives::block::Block]
     ColBlock = 2,
+    /// Column that stores Block headers.
+    /// - *Rows*: block hash (CryptoHash)
+    /// - *Content type*: [near_primitives::block_header::BlockHeader]
     ColBlockHeader = 3,
+    /// Column that stores mapping from block height to block hash.
+    /// - *Rows*: height (u64)
+    /// - *Content type*: block hash (CryptoHash)
     ColBlockHeight = 4,
+    /// TODO: this is something related to garbage collection - no idea yet.
     ColState = 5,
+    /// Mapping from BlockChunk to ChunkExtra
+    /// - *Rows*: BlockChunk (block hash + shard id)
```

Comment:
Should `+` here and elsewhere be taken as concatenation? Ie, we don't numerically add shard id to the hash, we concatenate the two, right? I wonder if there's obvious, short way to write that.

Reply:
done.